File: Find the Last Occurrence of the Number/find_last_occurrence.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    int x;
    printf("Enter the number to search: ");
    scanf("%d", &x);
    int index = -1;
    for (int i = n - 1; i >= 0; i--) {
        if (a[i] == x) {
            index = i;
            break;
        }
    }
    printf("Last occurrence index: %d\n", index);
    return 0;
}

File: Find the Last Occurrence of the Number/find_last_occurrence.py
n = int(input("Enter the number of elements: "))
a = []
for i in range(n):
    a.append(int(input(f"Enter element {i + 1}: ")))
x = int(input("Enter the number to search: "))
index = -1
for i in range(n - 1, -1, -1):
    if a[i] == x:
        index = i
        break
print("Last occurrence index:", index)
